masonry/src/theme.rs

```diff
@@ -1,9 +1,9 @@
 // Copyright 2019 the Xilem Authors and the Druid Authors
 // SPDX-License-Identifier: Apache-2.0
 
-//! Theme keys and initial values.
+//! Default values used by various widgets in their paint methods.
 
-#![allow(missing_docs)]
+#![allow(missing_docs, reason = "Names are self-explanatory.")]
```

Comment:
There are still some useful docs possible here; see https://docs.rs/color/0.2.3/color/palette/css/index.html.

Not to block on that for this change, but I'd maybe say:
```suggestion
// TODO: Docs as in https://docs.rs/color/0.2.3/color/palette/css/index.html.
#![allow(missing_docs, reason = "Names are reasonably self-explanatory.")]
```

Reply:
I'm not sure I'd even want these one-line docs. I'd be worried they could get out of sync with the actual values.

Reply:
I'm leaving that as-is unless you consider it a serious blocker.

Reply:
I don't consider this a blocker.